bundle/jsky.app.ot/src/main/java/jsky/app/ot/util/Resources.java

```diff
@@ -94,6 +96,19 @@ public static ImageIcon getIcon(String iconFileName) {
         return icon;
     }
 
+    public static void setOTFrameIcon(java.awt.Frame frame) {
+        if (Platform.get() != Platform.osx) {
+            // This has been reported to crash the JVM on OSX when the application
+            // is bundled as a DMG. Starting from the DMG The application fails to start
+            // without any visible indication on why it has failed
+            // However, we want to do this on Windows and Linux
```

Comment:
Ok, I updated the comment and reproduced the issue. It only happens if the app is in a DMG and it basically doesn't start, There are no messages on the screen or any logs. Clearly is the `setIconImage` call but I can't find a bug report matching this